examples/experimental/i3s-17-and-debug/geometry/cube-geometry.js

```diff
@@ -0,0 +1,89 @@
+import Geometry from '@luma.gl/engine/src/geometry/geometry';
+import {uid} from '@luma.gl/webgl';
+
+// prettier-ignore
+const CUBE_INDICES = new Uint16Array([
```

Comment:
Can't these geometries just be imported from luma.gl?

Reply:
Imported from luma